Metrics without Limits Beta Alert(s) (#6438)
DataDog/documentation: content/en/account_management/billing/custom_metrics.md

{{< img src="account_management/billing/custom_metrics/tracking_metric.mp4" alt="Tracking metric" video="true" responsive="true">}}

<div class="alert alert-warning">

> <strong>Metrics without Limits is now in public beta.</strong> <a href="/help">Contact Datadog support</a> for more information.

Author: ```suggestion
<strong>Metrics without Limits is now in public beta.</strong> Contact your CSM for more information.
```

For billing questions, contact your [Customer Success][25] Manager.

## Further reading

> {{< partial name="whats-next/whats-next.html" >}}[1]: /integrations

Reviewer: ```suggestion
{{< partial name="whats-next/whats-next.html" >}}

[1]: /integrations
```
